nasa/harmony-netcdf-to-zarr
# DAS-1708 - Fix names of images built locally.

## Description

This PR ensures that when a Docker image is built locally for the service its name matches the remote version (ghcr.io/nasa/harmony-netcdf-to-zarr). This is something that should have been caught when revamping the CI/CD - sorry about that!

I also have removed a script that installs the AWS CLI (primarily used to push images to AWS ECR) and the script that pushed images to AWS ECR, as these are no longer needed.

I think there's more clean-up that can be done here, but wanted to unblock local development first.

## Jira Issue ID

DAS-1708.

## Local Test Steps

* Pull this branch.
* Make a change to the NetCDF-to-Zarr service that you will be able to detect in your local Harmony instance (e.g., a logging statement).
* Run `bin/build-image`.
* Use `bin/restart-services` in the Harmony repo to pull in your new NetCDF-to-Zarr image. It should work.
* Issue a NetCDF-to-Zarr service request and look for the change you made in the second step.

## PR Acceptance Checklist
* ~~Tests added/updated and passing.~~
* ~~Documentation updated (if needed).~~
* ~~`version.txt` and `CHANGELOG.md` updated (if publishing a new release).~~
